RocketMan/zookeeper
# Ericg/162 add spin time element

Add ability for user to set spin time when adding tracks to the playlist if the show is "not live". This simplifies the process of creating playlists for shows that are generated at home for later broadcast. Changes include the following:

* move insert time generation into ui/Playlist.php
* informative user message when insert track fails at DB layer
* fix ajax error message handling for 500 errors
* show spin time input element when working on non-live playlists
* remove edate advance when time is change because it advanced date when it shouldn't
* abort track insert if spin time is less than most recent spin time

Test Cases:
* add all playlists types for live and non-live playlists
* add separator
* confirm informative user message if spin time is outside show time range
* confirm informative user message if spin time < latest track time
* confirm DnD of non-live playlist elements
* confirm created field is NULL for added spins when user does not enter a time value
